newspaper/text.py

```diff
@@ -156,3 +156,28 @@ def get_stopword_count(self, content):
         ws.set_stopword_count(len(overlapping_stopwords))
         ws.set_stop_words(overlapping_stopwords)
         return ws
+
+
+class StopWordsHindi(StopWords):
+    """Hindi segmentation
+    """
+    def __init__(self, language='hi'):
+        super(StopWordsHindi, self).__init__(language='hi')
+
+    def get_stopword_count(self, content):
```

Comment:
Also I'm not sure if Hindi is a language that simply tokenizes words by spaces (like English) or needs more complex tokenizing like Arabic/Chinese.

See: https://github.com/taranjeet/hindi-tokenizer. Do you have any advice? (I don't know Hindi at all)

Reply:
I am new to NLP. Will have a look into this library!